File: 009 - PalindromeNumber/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
        String s = String.valueOf(x);
        int left = s.length()-1;
        for (int right = 0; right < s.length(); right++){
            if (s.charAt(right)!=s.charAt(left)){
                return false;
            }
            left--;
        }
        return true;
    }
}

File: 009 - PalindromeNumber/Solution.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        num = str(x)
        right = len(num)-1
        for left in range(0, len(num)):
            if (num[left]!=num[right]):
                return False
            right-=1
        return True
